errai-cdi/jboss/src/main/java/org/jboss/errai/cdi/server/gwt/JBossLauncher.java

```diff
@@ -45,13 +49,17 @@
   public ServletContainer start(TreeLogger treeLogger, int port, File appRootDir) throws BindException, Exception {
     logger = new StackTreeLogger(treeLogger);
 
-    logger.branch(Type.INFO, "Starting launcher...");
+    logger.branch(Type.INFO, "Server launcher starting..");
 
     // Get properties
-    final String DEBUG_PORT = System.getProperty(JBOSS_DEBUG_PORT_PROPERTY, "8001");
-    final String TEMPLATE_CONFIG_FILE = System.getProperty(TEMPLATE_CONFIG_FILE_PROPERTY, "standalone-full.xml");
+    final String DEBUG_PORT 		     = System.getProperty(JBOSS_DEBUG_PORT_PROPERTY, "8001");
```

Comment:
I see you've added whitespace here to align the equals signs, but this line is off by a couple spaces. This isn't something we typically do in Errai code so it might be best to remove this formatting.